### PR TITLE
finished the ui of the admin page

### PR DESCRIPTION
**Pull request by sameerav**
review @amillman ?

note: the page doesn't do anything except exist and show itself when the current user is an admin

### CODE REVIEW

File: sage-ios/SAGE/SAGE/RootTabBarController.swift [hunk 1]
```diff
@@ -42,7 +43,20 @@ class RootTabBarController: UITabBarController, UINavigationControllerDelegate {
         let myStatsViewController = UIViewController()
         let profileViewController = UIViewController()
         
-        let rootViewControllers = [announcementsViewController, checkInViewController, myStatsViewController, profileViewController]
+        var rootViewControllers = [announcementsViewController, checkInViewController, myStatsViewController, profileViewController]
+        
+        // User.currentUser!.role = User.UserRole.Admin
+        // comment in the above line3 to see the admin page
+        if let role = User.currentUser?.role {
+            if role == User.UserRole.Admin {
```
Comment (amillman) [inline sage-ios/SAGE/SAGE/RootTabBarController.swift]: you can shorthand to .Admin right?

Comment (sameerav) [inline sage-ios/SAGE/SAGE/RootTabBarController.swift]: ya

**Comment by sameerav** [comment]
pushed changes @amillman !

**Comment by amillman** [comment]
is it posisble to make the icons slightly smaller? and make sure the title font conforms to the custom SAGE font, or else itll be Helvetica Neue in iOS8-

**Comment by sameerav** [comment]
made the icons 23px and made the font system font @amillman

**Comment by charlesx2013** [comment]
lgtm

**Pull request merged**